promptsource/seqio_tasks/tasks.py

```diff
@@ -37,6 +37,18 @@
     ("tweet_eval", "irony"),
 ]
 
+
+def strip_whitespace(output_or_target, example=None, is_target=False):
+    """
+    Temporary hack. Cached tasks from promptsource all have a leading space
+    on the ground-truth targets.
+    """
+    if is_target:
+        return output_or_target.strip()
+    else:
+        return output_or_target
```

Comment:
I think it's probably fair/correct to strip() both inputs and targets.. WDYT?

Reply:
Sure. We can double check later but can definitely strip both to be safe for now.